Makefile

```diff
@@ -178,6 +178,9 @@ KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
 	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v3.8.7)
 
+kustomize-v3.8.8: ## Download kustomize locally
```

Comment:
Why not just update kustomize version above, and use the same across all kustomize requirements in the build?

The `make deploy-[dr-cluster|hub]` will fail for those that have older kustomize binary and can be corrected on their local systems?

Reply:
That's right, if a version is already installed, i.e. `[ -f $(shell pwd)/bin/kustomize ]`, then a new version won't be installed.  So other solutions include deleting the existing one.  I'm not aware of a method that does version checking and even if there were one, `kustomize version` failed to output a version for me.  A workaround I thought of was parsing the version from `strings ~/ramen/bin/kustomize|grep ^mod\     sigs.k8s.io/kustomize/kustomize/v` but that may take about the same time as downloading a new one and I'm not sure how reliable it is with different versions.

Reply:
Delete solutions include:
- automate it when `deploy-[hub|dr-cluster]` encounters any non-zero return value and
  - trying it once more somehow, or
  - exiting and letting issuer retry
- a git log note in this commit instructing user to delete it
- others?

Thoughts @ShyamsundarR?

Reply:
Let dev setups fail! that is the solution, and over a little time, they will come to realize that kustomize version has to change. Hence my solution is do nothing :)

Auto builders will not have the issue, as they would start with a clean system.

Reply:
IOW update existing `kustomize` rule for now... and leave it at that.

Reply:
Benamar just encountered the error from kustomize v3.8.7, here it is for reference:
```sh
/home/user1/go/src/github.com/ramen/bin/controller-gen "crd:trivialVersions=true,preserveUnknownFields=false" rbac:roleName=operator-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
cd config/hub/manager && /home/user1/go/src/github.com/ramen/bin/kustomize edit set image controller=quay.io/ramendr/ramen-operator:latest
/home/user1/go/src/github.com/ramen/bin/kustomize build --load_restrictor none config/hub/default | kubectl apply -f -
Error: 2 errors occurred:
	* accumulateFile error: "accumulating resources from 'apiVersion: builtin\nkind: LabelTransformer\nmetadata:\n  name: temporary\nlabels:\n  app: ramen-hub\nfieldSpecs:\n- kind: Deployment\n  path: metadata/labels\n- kind: Deployment\n  path: spec/selector/matchLabels\n- kind: Deployment\n  path: spec/template/metadata/labels\n- kind: Service\n  path: metadata/labels\n- kind: Service\n  path: spec/selector': open /home/user1/go/src/github.com/ramen/config/hub/default/apiVersion: builtin\nkind: LabelTransformer\nmetadata:\n  name: temporary\nlabels:\n  app: ramen-hub\nfieldSpecs:\n- kind: Deployment\n  path: metadata/labels\n- kind: Deployment\n  path: spec/selector/matchLabels\n- kind: Deployment\n  path: spec/template/metadata/labels\n- kind: Service\n  path: metadata/labels\n- kind: Service\n  path: spec/selector: no such file or directory"
	* loader.New error: "error loading apiVersion: builtin\nkind: LabelTransformer\nmetadata:\n  name: temporary\nlabels:\n  app: ramen-hub\nfieldSpecs:\n- kind: Deployment\n  path: metadata/labels\n- kind: Deployment\n  path: spec/selector/matchLabels\n- kind: Deployment\n  path: spec/template/metadata/labels\n- kind: Service\n  path: metadata/labels\n- kind: Service\n  path: spec/selector with git: url lacks host: apiVersion: builtin\nkind: LabelTransformer\nmetadata:\n  name: temporary\nlabels:\n  app: ramen-hub\nfieldSpecs:\n- kind: Deployment\n  path: metadata/labels\n- kind: Deployment\n  path: spec/selector/matchLabels\n- kind: Deployment\n  path: spec/template/metadata/labels\n- kind: Service\n  path: metadata/labels\n- kind: Service\n  path: spec/selector, dir: evalsymlink failure on '/home/user1/go/src/github.com/ramen/config/hub/default/apiVersion: builtin\nkind: LabelTransformer\nmetadata:\n  name: temporary\nlabels:\n  app: ramen-hub\nfieldSpecs:\n- kind: Deployment\n  path: metadata/labels\n- kind: Deployment\n  path: spec/selector/matchLabels\n- kind: Deployment\n  path: spec/template/metadata/labels\n- kind: Service\n  path: metadata/labels\n- kind: Service\n  path: spec/selector' : lstat /home/user1/go/src/github.com/ramen/config/hub/default/apiVersion: builtin\nkind: LabelTransformer\nmetadata:\n  name: temporary\nlabels:\n  app: ramen-hub\nfieldSpecs:\n- kind: Deployment\n  path: metadata: no such file or directory, get: invalid source string: apiVersion: builtin\nkind: LabelTransformer\nmetadata:\n  name: temporary\nlabels:\n  app: ramen-hub\nfieldSpecs:\n- kind: Deployment\n  path: metadata/labels\n- kind: Deployment\n  path: spec/selector/matchLabels\n- kind: Deployment\n  path: spec/template/metadata/labels\n- kind: Service\n  path: metadata/labels\n- kind: Service\n  path: spec/selector"
```
It was resolved by removing the downlevel kustomize (v3.8.7) and running a command to download the new one (v3.8.8):
```sh
rm bin/kustomize
make kustomize
```